### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by ravikumar0403**
Change Log:
- add input components
- add image components
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -369,6 +371,139 @@ <h3>Disabled button</h3>
                     </code>
                 </pre>
             </div>
+
+            <!-- Image component starts -->
+            <div id="image">
+                <h2>Images</h2>
+                <p>Images can be responsive to fit the parent's width, and also can be customised to have rounded corners or as a circle</p>
+                <h3>Responsive Image</h3>
+                <p>You can add class <span class="doc-classname">img-responsive</span> to make your image fit the width of container. It's height will get adjusted by keeping the aspect ratio same.</p>
```
Comment (yashpatel1797) [inline index.html]: Here class name is confusing if possible make a change.

**Review by yashpatel1797 — COMMENTED** [review]
The code looks good.

**Review by nyyshaaa — COMMENTED** [review]
in email section should it show invalid by default? all else looks good.

**Comment by ravikumar0403** [comment]
> in email section should it show invalid by default? all else looks good.

invalid class can be added and removed dynamically using javascript

File: index.html (hunk 1 above)
Comment (ravikumar0403) [reply index.html]: will change this

**Pull request merged**